tiny_dnn/activations/activation_function.h

```diff
@@ -161,7 +161,7 @@ class softmax : public function {
   vec_t df(const vec_t &y, size_t index) const override {
     vec_t v(y.size(), 0);
     for (size_t i = 0; i < y.size(); i++)
-      v[i] = (i == index) ? df(y[index]) : -y[i] * y[index];
+      v[i]        = (i == index) ? df(y[index]) : -y[i] * y[index];
```

Comment:
@edgarriba @Randl I will put curly braces from next time.

I used clang-format for different program today and saw a similar problem. I ended up using `AlignConsecutiveAssignments: false` to prevent it.

Some of the other options I changed from tiny-dnn's `.clang-format` are like these.
```
ReflowComments:  false
BreakStringLiterals: false
SortIncludes:    false
```

Though clang-format is pretty useful, I will not post an essay titled "How I learned to stop worrying and love the clang-format". :persevere: